monitoring_service.rst

```diff
@@ -42,157 +30,62 @@ Information Flow
-for a high level of reliability and the client knows neither how many MSs are monitoring his channel nor how reliable
-they are. These tradeoffs are made to favor simplicity of implementation.
-
-Privacy
--------
-
-The recipient and the actual transferred amounts are hidden by providing a hashed balance proof (or state). This
-provides some sort of privacy even if it can potentially be recalculated.
-
-Security Analysis (inspired by PISA)
-====================================
-
-State Privacy
--------------
-
-Blinded BPs are published to the MS as part of the Monitor Request in the matrix room and then submitted to the smart
-contract.
-
-Fair Exchange
--------------
-
-Clients can freely choose the reward for the MS, so it is easy for him to choose the amount in a way that makes the
-exchange attractive for himself. The client can’t know if a MS started monitoring his payment channel, so he can’t use
-such feedback to arrive at a reward where he knows that the deal is attractive for both him and the MS. Neither can he
-recognize if there is no such possible reward.
-The MS on the other hand can freely choose to ignore requests when the reward is too low, so he will only choose
-requests that he deems fairly rewarded. If the MS ignores the client’s request, the client keeps his deposit and it can
-be used by other MSs or for later BPs. In summary, the exchange is fair for both parties, but there is a high likelihood
-that no exchange will happen at all.
-
-Non-frameability
-----------------
-
-MSs can put the clients channel deposit at risk by ignoring all client requests. But since a MS can’t force other MSs to
-ignore client requests, this can not be considered as framing. When only a single MS is monitoring the channel, the MS’s
-dispute intervention and the reward payment happen atomically inside the SC. In this case, no party can frame the other.
-
-When multiple MSs try to settle the same dispute, only the first one doing so receives a reward, but all of them have to
-invest resources to monitor the channel and spend gas to interact with the SC. If you find a way to continuously front
-run other MSs, you can drain their resources and block their only income. However, while doing so you fulfilled the MS’s
-duty to settle the payment channel correctly and protect the client’s deposit.
-In the short run, this is an acceptable outcome for the client. In the long run, this will drive other MSs out of
-business and thus reduce redundancy and reliability of the overall MS ecosystem. Since all MSs try to be the first to
-submit a BP, it is unlikely that a single MS will continuously be the fastest, but slightly slower MSs will still not
-get any rewards even if they are well behaved and reliable.
+The MS can claim its reward after successfully submitting its client’s balance proof update. This is only allowed when the Monitoring Service is registered in the Service Registry. For more infos see the :ref:`ServiceRegistry` contract.
```

Comment:
`claim()` works even after the MS's registration expires.

`monitor()` requires a valid registration.